2.13/tests/test/configs/config_mock.json

```diff
@@ -3,106 +3,54 @@
                 },
-                "organization_guid": "org-guid-here",
-                "space_guid": "space-guid-here" 
-            },            
-            "scenario": "new",
-            "notes": "Fake service instance",
-            "async": true
-        },
-        {
-            "instance_id":"EXISTING_ID",
-            "body": {
-                "plan_id": "d3031751-XXXX-XXXX-XXXX-a42377d3320e", 
-                "service_id" :"acb56d7c-XXXX-XXXX-XXXX-feb140a59a66",                                
-                "parameters": {
-                    "billing-account": "fake-account"
-                },
-                "organization_guid": "org-guid-here",
-                "space_guid": "space-guid-here" 
-            },            
-            "scenario": "new",
-            "notes": "Fake service instance",
-            "async": true
-        },
-        {
-            "instance_id":"EXISTING_ID",
-            "body": {
-                "plan_id": "0f4008b5-XXXX-XXXX-XXXX-dace631cd648", 
-                "service_id" :"acb56d7c-XXXX-XXXX-XXXX-feb140a59a66",                                
-                "parameters": {
-                    "billing-account": "fake-account"
+                {
+                    "operation": "update",
+                    "plan_id": "0f4008b5-XXXX-XXXX-XXXX-dace631cd648",
+                    "async": true,
+                    "parameters": {
+                        "parameter1":1,
+                        "parameter2":"foo"
+                    }
                 },
-                "organization_guid": "org-guid-here",
-                "space_guid": "space-guid-here" 
-            },            
-            "scenario": "conflict",
-            "notes": "Fake service instance",
-            "async": true
-        }
-    ],
-    "updates":[
-        {
-            "instance_id":"",
-            "body":{
-                "context" :{
-                    "platform": "cloudfoundry",
-                    "some_field": "some-contextual-data"
+                {
+                    "operation": "bind",
+                    "parameters": {
+                        "parameter1-name-here": 1,
+                        "parameter2-name-here": "parameter2-value-here"
+                    }
                 },
-                "service_id": "2e2fc314-37b6-4587-8127-8f9ee8b33fea",
-                "plan_id":"6ddf6b41-fb60-4b70-af99-8ecc4896b3cf",
-                "parameters":{
-                    "parameter1":1,
-                    "parameter2":"foo"
+                {
+                    "operation": "unbind"
                 },
-                "previous_values":{
-                    "plan_id":"6ddf6b41-fb60-4b70-af99-8ecc4896b3cf",
-                    "service_id":"2e2fc314-37b6-4587-8127-8f9ee8b33fea",
-                    "organization_id":"org-guid-here",
-                    "space_id":"spcae-guid-here"
+                {
+                    "operation": "deprovision",

```

Comment:
According to the spec, I'm wondering if it's better to add `plan_id` field here.

Reply:
Let me clarify that, the config file no longer directly specifies the request body. The test framework is able to what the `plan_id` is currently and build the request body.